test/linting/lint.sh

```diff
@@ -104,73 +90,40 @@ function _hadolint
 function _shellcheck
 {
   local SCRIPT='SHELLCHECK'
-  local ERR=0
-  local LINT=(shellcheck -x -S style -Cauto -o all -e SC2154 -W 50)
-
-  if ! __in_path "${LINT[0]}"
-  then
-    __log_failure 'linter not in PATH'
-    return 2
-  fi
+  
+  # File paths for shellcheck:
+  F_SH="$(find . -type f -iname '*.sh' \
+    -not -path './test/bats/*' \
+    -not -path './test/test_helper/*' \
+    -not -path './target/docker-configomat/*' -print0 \
+    | xargs -0
+  )"
+  F_BIN="$(find 'target/bin' -perm +111 -type f -or -type l)"
+  F_BATS="$(find 'test' -maxdepth 1 -type f -iname '*.bats')"
+
+  # This command is a bit easier to grok as multi-line. There is a `.shellcheckrc` file, but it's only supports half of the options below, thus kept as CLI:
+  # shellcheck disable=SC2206
+  CMD_SHELLCHECK=(shellcheck 
+    --external-sources 
+    --check-sourced
+    --severity=style
+    --color=auto
+    --wiki-link-count=50
+    --enable=all
+    --exclude=SC2154
+    --source-path=SCRIPTDIR
+    ${F_SH} ${F_BIN} ${F_BATS}
+  )
```

Comment:
```suggestion
  CMD_SHELLCHECK=(shellcheck 
    --external-sources 
    --check-sourced
    --severity=style
    --color=auto
    --wiki-link-count=50
    --enable=all
    --exclude=SC2154
    --source-path=SCRIPTDIR
    "${F_SH} ${F_BIN} ${F_BATS}"
  )
```

Reply:
The quotes variables doesn't work

Reply:
I wasn't able to suggest both changes as a single change. You did test this `CMD_SHELLCHECK` suggestion with the `docker run` change that removed quotes for `CMD_SHELLCHECK[@]` right? Otherwise no it would not work.

Reply:
Doesn't work on ubuntu either
<img width="950" alt="Screen Shot 2021-06-06 at 9 03 56 PM" src="https://user-images.githubusercontent.com/5896030/120946564-e9244200-c70a-11eb-8ad9-b5dc1105e88f.png">

Reply:
> Doesn't work on ubuntu either

It absolutely does when applying both of my change requests. Which I think you figured out based on your related comment?

Reply:
Yep, we're good :)